orchagent/portsorch.cpp

```diff
@@ -2563,13 +2566,22 @@ void PortsOrch::doLagTask(Consumer &consumer)
                 }
                 else if (fvField(i) == "oper_status")
                 {
-                    if (fvValue(i) == "down")
+                    operation_status = fvValue(i);
+                    if (!string_oper_status.count(operation_status))
                     {
-                        gNeighOrch->ifChangeInformNextHop(alias, false);
+                        SWSS_LOG_ERROR("Invalid operation status value:%s", operation_status.c_str());
+                        it++;
+                        continue;
                     }
-                    else
+
+                    gNeighOrch->ifChangeInformNextHop(alias,
+                                                      (operation_status == "down") ?
+                                                      false : true);
+                    Port lag;
+                    if (getPort(alias, lag))
                     {
-                        gNeighOrch->ifChangeInformNextHop(alias, true);
+                        operation_status_changed = (string_oper_status.at(operation_status) != lag.m_oper_status) ?
+                                                   true : false;
```

Comment:
> true : false; [](start = 50, length = 14)

remove

Reply:
Sorry I dint get what to remove. Could you please elaborate?

Reply:
Remove  `? true : false`. The original bool statement is good enough.

---
In reply to: [433554653](https://github.com/Azure/sonic-swss/pull/1242#discussion_r433554653) [](ancestors = 433554653)